website/docs/reference/dbt-jinja-functions/source.md

```diff
@@ -13,9 +13,11 @@ select * from {{ source(source_name, table_name) }}
 
 This function:
 - Returns a [Relation](/reference/dbt-classes#relation) for a [source](/docs/build/sources)
+  - Note &mdash; Starting from dbt v1.6 or higher, `{{ {source.identifier }}` will fetch the identifier (such as table name). Previously it would only return the request.
```

Comment:
```suggestion
```
Unless you validated this in the IDE, I don't think it's accurate for sources, because those tables _always_ exist